arch/arm/src/common/Toolchain.defs

```diff
@@ -233,7 +233,11 @@ else
   # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=105523
 
   ifeq ($(GCCVER),)
-    export GCCVER := $(shell $(CC) --version | grep gcc | sed -r 's/.* ([0-9]+\.[0-9]+).*/\1/')
+    ifeq ($(CONFIG_WINDOWS_NATIVE),y)
+      export GCCVER := $(shell $(CC) --version | grep gcc | sed -r "s/.* ([0-9]+\.[0-9]+).*/\1/")
```

Comment:
double quotes is also applicable to linux

Reply:
Done!